src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -39,12 +40,30 @@ public Moves getMove() {
         } catch (IOException e) {
             e.printStackTrace();
         }
-        if (userMove.equals("rock") || userMove.equals("paper") || userMove.equals("scissors")) {
-            return convertMove(userMove);
+        return getConvertedMove(userMove);
+    }
+
+    public Moves getMoveTwo(String gameMode) {
```

Comment:
From the name I infer that there is only a limited number of game modes? Maybe an `enum GameMode { ... }` would be more fitting? Paging Dr. @gabbendorf

Reply:
Also, if we were to elevate `gameMode` above a String, maybe we could have the gameMode know how to play differently so you don't need the if-statement here?